includes/wc-functions.php

```diff
@@ -351,10 +352,10 @@ function ( $value ) {
             $files = [];
 
             $file_names    = isset( $data['_wc_file_names'] ) ? array_map( 'wc_clean', $data['_wc_file_names'] ) : [];
-            $file_urls     = isset( $data['_wc_file_urls'] ) ? array_map( 'esc_url_raw', array_map( 'trim', $data['_wc_file_urls'] ) ) : [];
```

Comment:
what's the reason behind removing the isset check?

Reply:
`isset( $data['_wc_file_urls'] )` This is checked earlier in the execution flow.